lib/glific/erp.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Glific.ERP do
+  @moduledoc """
+  ERP API integration utilities for fetching organization data.
+  """
+
+  require Logger
+  use Tesla
+
+  @erp_base_url "https://t4d-erp.frappe.cloud/api/resource"
+
+  @client Tesla.client([
+            {Tesla.Middleware.JSON, engine_opts: [keys: :atoms]},
+            Tesla.Middleware.FollowRedirects
```

Comment:
because the curl request for fetching the name and id is `curl -L -X GET "https://t4d-erp.frappe.cloud/api/resource/Customer/?fields=%5B%22name%22,%22customer_name%22%5D&limit_page_length=0"` and if i dont use rediect it was giving error that the url is redirected to a new location.

Reply:
ok, thanks.